common/lib/client/auth.js

```diff
@@ -239,8 +239,15 @@ var Auth = (function() {
 				}
 			}
 			tokenRequestCallback = function(params, cb) {
-				var authHeaders = Utils.mixin({accept: 'application/json'}, authOptions.authHeaders);
-				Http.getUri(rest, authOptions.authUrl, authHeaders || {}, Utils.mixin(params, authOptions.authParams), function(err, body, headers, unpacked) {
+				var authHeaders = Utils.mixin({accept: 'application/json'}, authOptions.authHeaders),
+						authParams = Utils.mixin(params, authOptions.authParams);
+				Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Sending; ' + authOptions.authUrl + '; Params: ' + JSON.stringify(authParams));
+				Http.getUri(rest, authOptions.authUrl, authHeaders || {}, authParams, function(err, body, headers, unpacked) {
+					if (err) {
+						Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Received Error; ' + JSON.stringify(err));
+					} else {
+						Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Received; body: ' + JSON.stringify(BufferUtils.isBuffer(body) ? body.toString() : body));
```

Comment:
> I am not sure what object the body will aways be

Either a buffer or string at this point. If it's neither then it's a bug elsewhere.

> What is the downside?

If we know it's not necessary it should be removed.